src/layouts/App.scss

```diff
@@ -457,13 +457,6 @@ body {
   background-color: rgba(0,0,0,.4);
 }
 
-.modal {
-  position: absolute;
```

Comment:
this doesnt mess up other modals?

Reply:
No, per the PR description, it fixes them